lib/still/compiler/incremental/node.ex

```diff
@@ -124,11 +124,23 @@ defmodule Still.Compiler.Incremental.Node do
         source_file: %Still.SourceFile{input_file: state.file, run_type: :compile}
       }
 
+      Logger.error(error)
       ErrorCache.set({:error, error})
+
+      if Application.get_env(:still, :compiling) do
```

Comment:
We can probably abstract this via the `config` function. Also feels better to give it a `compiling?/0` name, rather than leaking `Application.get_env/2` details into these random modules.